pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -13,6 +13,10 @@
   campaign_date_year: "2017"
   campaign_date_full_year: "December 4-10, 2017"
 
+  cookie_banner_message: 'By continuing to browse our site or clicking "I agree," you agree to the storing of cookies on your computer or device.'
+  cookie_banner_more_information: "See Code.org's Privacy Policy."
+  cookie_banner_accept: 'I agree'
```

Comment:
Weird - why is this i18n file structured differently than the dashboard one?

Reply:
Not sure about the history here.  Maybe @tanyaparker knows?

Reply:
By structured differently do you mean not that much nesting with the keys? We don't really have a protocol for how to namespace. The pattern just naturally developed and we prepended related strings with a common term.

Reply:
Yeah the nesting difference - this ends up requiring some special-case code elsewhere in the PR, which is sort of a bummer.

Reply:
Ah! Yes, I wouldn't be opposed to structuring the files to be more like dashboard loc files.